packages/forma-36-react-components/src/components/Switch/style.css

```diff
@@ -0,0 +1,56 @@
+@import 'settings/colors';
+
+.Switch__wrapper {
+  display: flex;
+}
+
+.Switch {
+  width: 35px;
```

Comment:
Please use rems for dimensions. We use `calc` and a formula for translating pixel values to rem

**Example to render 35px to equivalent rem value based on our font base size:**

```css
calc(1rem * (35/ var(--font-base-default)));
```